assets/js/script.js

```diff
@@ -127,6 +127,61 @@ function form() {
   var emailInputElement = document.querySelector('#email');
   var messageInputElement = document.querySelector('#message');
   var errorMessagesElement = document.querySelector('.error-messages');
```

Comment:
- I think it would be better to use const and let to declare your variables , Kindly check [This link ](https://medium.com/nerd-for-tech/var-let-and-const-in-javascript-15e41cf90f01) to learn more about the difference between `var`, `let` and `const`

Reply:
These are the compiled version of the file due to compatibility. It's automatically generated. The code is located on the src folder.